rules/rules-reviewed/azure/springboot/spring-boot-to-azure-config-server.windup.xml

```diff
@@ -43,5 +43,27 @@
                 <matches pattern="spring\.config\.import|spring\.cloud\.config\.uri"/>
             </where>
         </rule>
+        <rule id="spring-boot-to-azure-config-server-02000">
+            <when>
+                <or>
+                    <project>
+                        <artifact groupId="org.springframework.cloud" artifactId="spring-cloud-config-server"/>
+                    </project>
+                    <javaclass references="org.springframework.cloud.config.server.EnableConfigServer.EnableConfigServer">
+                        <location>IMPORT</location>
+                    </javaclass>
+                </or>
+            </when>
+            <perform>
+                <hint title="Config Server found" category-id="information" effort="0">
+                    <message>
+                        The application uses Spring Cloud Config Server.
```

Comment:
how about we enrich this message to tell customers to configure the config server in Azure Spring Apps instance and see [Set up a Spring Cloud Config Server instance for your service](https://learn.microsoft.com/en-us/azure/spring-apps/how-to-config-server) for more info